Library/Formula/asterisk.rb

```diff
@@ -0,0 +1,27 @@
+class Asterisk < Formula
+  desc "Toolkit for building communications applications"
+  homepage "http://www.asterisk.org/"
+  url "http://downloads.asterisk.org/pub/telephony/asterisk/asterisk-13.6.0.tar.gz"
+  sha256 "8a01b53c946d092ac561c11b404f68cd328306d0e3b434a7485a11d4b175005a"
+
+  option "with-sample-config", "Install the sample config files.  NOTE. Without this, you won't have any config file."
```

Comment:
I'm not yet so familiar with asterisk as to knowledgable answer that question.

As a base for this formula I used the last one that was removed in 29680bac3a4864a0a7207fe2ce40c2c0d90c06ec, that is wehre `with-sample-config` option stems from.

I would be ok with removing it.